virtcontainers/clh.go

```diff
@@ -0,0 +1,1260 @@
+// Copyright (c) 2019 Ericsson Eurolab Deutschland GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package virtcontainers
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	persistapi "github.com/kata-containers/runtime/virtcontainers/persist/api"
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/kata-containers/runtime/virtcontainers/device/config"
+	"github.com/kata-containers/runtime/virtcontainers/store"
+	"github.com/kata-containers/runtime/virtcontainers/types"
+	"github.com/kata-containers/runtime/virtcontainers/utils"
+)
+
+//
+// Constants and type definitions related to cloud hypervisor
+//
+
+type clhState uint8
+
+const (
+	clhNotReady clhState = iota
+	clhReady
+)
+
+const (
+	clhTimeout            = 10
+	clhSocket             = "clh.sock"
+	vfsdSocket            = "virtiofsd.sock"
```

Comment:
Could we please rename that one to `virtioFsSocket`, please? I get confused with the vfs prefix as in VFS...

Reply:
changed